tests/config/test_data_config.py

```
@@ -0,0 +1,104 @@
+    config = DataConfig(train_labels_path="train.slp", val_labels_path="val.slp")
+    assert config.provider == "LabelsReader"
+    assert config.train_labels_path == "train.slp"
+    assert config.val_labels_path == "val.slp"
+    assert config.user_instances_only is True
+    assert config.chunk_size == 100
+
+
+def test_data_config_missing_values():
+    # Test that DataConfig raises an error if required fields are missing.
+    # with pytest.raises(ValidationError):
+    config = OmegaConf.structured(DataConfig())
```

Ruff (0.8.2)

Local variable `config` is assigned to but never used

Remove assignment to unused variable `config`

(F841)